app/views/prior_authority/steps/reason_why/edit.html.erb

```diff
@@ -0,0 +1,56 @@
+<%= javascript_import_module_tag 'file-upload' %>
+
+<% title t('.page_title') %>
+<% decision_step_header %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary(@form_object) %>
+    <span class="govuk-caption-xl"><%= t(".caption") %></span>
+    <%= step_form @form_object do |form| %>
+      <%= form.govuk_text_area :reason_why,  max_chars: 2000, rows: 16, label: { size: 'xl', text: t(".page_title") }, hint: { text: t('.reason_why_hint_html')}  %>
+
+      <h1 class="govuk-heading-m"><%= t(".supporting_documentation") %></h1>
+      <span class="govuk-caption-m govuk-!-margin-bottom-8"><%= t(".supporting_documentation_caption") %></span>
+
+      <div class="moj-multi-file-upload">
+        <div class="moj-multi-file-upload__upload">
+          <%= form.govuk_file_field :documents, label: { text: t('.file_upload.heading'), size: 'm' }, class: 'moj-multi-file-upload__input', multiple: true, accept: "application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document,application/rtf,image/jpeg,image/bmp,image/png,image/tiff,application/pdf" %>
```

Comment:
Is there any way of reusing ReasonWhyController::SUPPORTED_FILE_TYPES here? I feel like at some point otherwise we'll scratch our heads why a particular file type looks like it should be supported but isn't when the list changes.

Reply:
extraced to a common PORO so easier to use in required places